.env.example

```diff
@@ -14,8 +14,7 @@ KODI_PORT=
 KODI_USERNAME=
 KODI_PASSWORD=
 
-SKILL_APPID=
-SKILL_VERIFY_CERT=
```

Comment:
Well you know more than I do about the verification stuff. I haven't done anything with it. I would have to assume though that flask-ask takes care of the cert verification. Although I wasn't aware of it until you mentioned it.

I'm manually adding the APPID verification if the env variable exists though.